README.md

```diff
@@ -118,9 +118,10 @@ Optional - Create a kubernetes generic secret that defines authentication for th
 The name of the secret is `${NAME}-authentication` e.g. `buses-em-authentication`.
 
 For signature verification of POST, PUT, PATCH requests create the property `signature-secret` with the value of secret.
-(See #8 for details of signature).
 
-<img width="394" alt="image" src="https://user-images.githubusercontent.com/3769612/65935654-b6229a80-e3ce-11e9-92ff-a13ace0f7cf6.png">
+Click here to see details on [enabling signature authentication](https://github.com/IBMStreams/endpoint-monitor/blob/master/docs/signature_auth.md)
```

Comment:
Can you clean up the section so that all the details are in the sub-page and not in both places, e.g. line 120 is duplicating info. 

Basically line 120 should be an overview of what authentication is being added, and then defer to the subpage (line 122) for the details.